### PR TITLE
Replaced generic graphic with Hacktoberfest graphic and added alt text

### PR DESCRIPTION
**Pull request by bartkalanski**
**Description**

Issue #884 "[Programs] Add Layer5 Community Hacktoberfest logo to program page"

This PR replaces generic graphic with Hacktoberfest graphic and adds alt text

Signed-off-by Bart Kalanski bkalanski@gmail.com
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: collections/_programs/hacktoberfest.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 permalink: /programs/hacktoberfest
 excerpt: "Layer5 - Hacktoberfest"
 ---
-<img src="/assets/images/programs/hacktoberfest2020/hacktober.svg" class="image-center" />
+<img src="https://user-images.githubusercontent.com/7570704/94851172-b1336500-03ed-11eb-917c-9bdb67ba8684.png" alt="Hacktoberfest and Layer5 logo" class="image-center" />
```
Comment (Jashpatel1) [inline collections/_programs/hacktoberfest.html:11]: Can you download the graphic in assets folder and then add it? Also the height to width ratio in current preview looks disturbed.

Comment (bartkalanski) [reply collections/_programs/hacktoberfest.html:11]: No problem, that makes more sense!

Comment (bartkalanski) [reply collections/_programs/hacktoberfest.html:11]: The width for the image is set at 40%, I set it to 60% in chrome devtools and it does seem to look better. Would you say that's the best way of improving the issue that you mentioned ?

Comment (Jashpatel1) [reply collections/_programs/hacktoberfest.html:11]: I think @leecalcote and @chandrashritii can look into if we could have a wider graphic.

**Review by Jashpatel1 — DISMISSED** [review]
Nice work. Thanks @bartkalanski 
Can something be done with the image @leecalcote @chandrashritii ?

**Comment by bartkalanski** [comment]
> Nice work. Thanks @bartkalanski
> Can something be done with the image @leecalcote @chandrashritii ?

Thank you! Pleasure as always :)

**Comment by Nikhil-Ladha** [comment]
Also, in mobile view the image size can be increased a bit.

**Comment by leecalcote** [comment]
If the image is moved into the left column, this will solve the issue of too much whitespace (the image being much taller than it is wider). It will also solve part of the problem of the questionnaire feeling so tall vs the text on the left.

Screen Shot 2020-10-04 at 11 22 59 AM

@bartkalanski, would you mind relocating?

**Comment by bartkalanski** [comment]
> If the image is moved into the left column, this will solve the issue of too much whitespace (the image being much taller than it is wider). It will also solve part of the problem of the questionnaire feeling so tall vs the text on the left.
> 
> Screen Shot 2020-10-04 at 11 22 59 AM
> 
> @bartkalanski, would you mind relocating?

I'm happy to have a go at this, would you like to me to move the image into the left column only ? Or the hacktoberfest heading as well ?

**Comment by Jashpatel1** [comment]
> I'm happy to have a go at this, would you like to me to move the image into the left column only ? Or the hacktoberfest heading as well ?

The graphic would be enough I guess.

**Comment by Nikhil-Ladha** [comment]
I think the second heading `Hactoberfest 2020 with Layer5` can be removed or switched with the outer title and remove this one.
Also, if you could adjust the height of the iframe to be same as that of the left column having the paragraph.

**Comment by Nikhil-Ladha** [comment]
I don't see the iframe height adjusted to be same as that of the left column.
Also, the image size needs to be increased a bit for mobile view.

**Comment by Nikhil-Ladha** [comment]
Also, please resolve the merge conflict too

**Comment by leecalcote** [comment]
@Nikhil-Ladha we might be able to forego the iframe height update. I made the iframe this tall in order to fully display the questionnaire without the need for scrolling (so that the user doesn't have a scrollbar inside of another scrollbar). The text in the lefthand column is vertically-aligned to the top. 

If you consider that it's a better experience to have the iframe height adjust on the form, though, no sweat, that'll work as well.

**Comment by leecalcote** [comment]
@Nikhil-Ladha good call about the mobile view. We have an issue with the google form spilling off the page, too.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha good call about the mobile view. We have an issue with the google form spilling off the page, too.

Already opened an issue for that...and it's been worked upon by someone.🙂

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha we might be able to forego the iframe height update. I made the iframe this tall in order to fully display the questionnaire without the need for scrolling (so that the user doesn't have a scrollbar inside of another scrollbar). The text in the lefthand column is vertically-aligned to the top.
> 
> If you consider that it's a better experience to have the iframe height adjust on the form, though, no sweat, that'll work as well.

Just to omit the white space beneath the text. If that's intentional we may skip that. But I think it would look a bit odd.

**Comment by leecalcote** [comment]
Screen Shot 2020-10-05 at 11 10 13 AM

This whitespace, right? Given that the form is embedded in the div, unless we cut the form in half, we won't be able to be rid of the whitespace. Cutting the form in half will have the effect of removing the whitespace and making the user scroll on the form.

Sounds like your thinking is that this might be a better user experience. Let's go with your suggestion and reduce the height on the iframe. 👍

**Comment by bartkalanski** [comment]
> I don't see the iframe height adjusted to be same as that of the left column.
> Also, the image size needs to be increased a bit for mobile view.

My bad, I assumed you meant for it to be inline with the paragraph. Do you have any advice on how I could reduce the size of the iframe ?

**Comment by bartkalanski** [comment]
> Also, please resolve the merge conflict too

All done, but I checked the new preview and it's looking wrong

**Comment by bartkalanski** [comment]
> Screen Shot 2020-10-05 at 11 10 13 AM
> 
> This whitespace, right? Given that the form is embedded in the div, unless we cut the form in half, we won't be able to be rid of the whitespace. Cutting the form in half will have the effect of removing the whitespace and making the user scroll on the form.
> 
> Sounds like your thinking is that this might be a better user experience. Let's go with your suggestion and reduce the height on the iframe. 👍

I should be able to achieve this by setting height to 1405 and adding scrolling="yes" attribute

**Review by Jashpatel1 — APPROVED** [review]
As the spilling in mobile view is worked upon as a different issue, I guess this PR is complete.

**Review by leecalcote — APPROVED** [review]
Fantastic. :)

**Comment by bartkalanski** [comment]
I noticed I must’ve put the old hacktoberfest  graphic source code in the latest commit, which means the old graphic is showing instead of the new one, do you mind if I quickly fix that ?

**Comment by leecalcote** [comment]
Hi Bart, not at all. Please do.

**Pull request merged**